src/components/date-picker/DatePicker.tsx

```diff
@@ -128,6 +128,8 @@ export interface DatePickerProps extends WrappedComponentProps {
     isDisabled?: boolean;
     /** Is input required */
     isRequired?: boolean;
+    /** Enables pikaday's default keyboard input support */
+    isKeyboardInputAllowed?: boolean;
```

Comment:
Do we want to add a default prop value for this to stay consistent with `isTextInputAllowed`?

Reply:
Good catch, will do 👍